Assets/Plugins/iOS/Shopify/Checkout/Checkout.WebViewController.swift

```diff
@@ -56,28 +57,73 @@ class WebViewController: UIViewController {
     
     override func loadView() {
         super.loadView()
-        self.view = WebContentView(frame: view.frame)
+        self.view = WKWebView(frame: view.frame)
     }
     
     override func viewDidLoad() {
         super.viewDidLoad()
         webView.navigationDelegate = self
-        contentView.setActionButton(target: self, selector: #selector(didPressCancel))
-        
+        setupNavigationBar()
```

Comment:
The contents of this method sets up views. It should probably be part of `override func loadView()` instead.